scripts/bootstrap.sh.in

```diff
@@ -44,6 +46,16 @@ EOF
 
 exec > >(tee -ia "${LOGFILE}") 2>&1
 
+determine_os() {
+    # This way permit to determine All the recent OS like Centos, Ubuntu
+    # and Debian
+     if [ -f /etc/os-release ]; then
+         . /etc/os-release;
+         OS=$ID;
+         VER=$VERSION_ID;
```

Comment:
This variable does not seem to be used anywhere.

Reply:
Indeed, I put it in case of we support several version of one OS, it permit to sort by OS version, but it's not accurate for now... Should I deleted it ?

Reply:
If it's not useful in any way, it should be deleted, yes.
We will add it back the day we'll need it... if this day comes.

Reply:
I deleted it.